vxquery-cli/src/main/java/org/apache/vxquery/cli/VXQuery.java

```diff
@@ -120,200 +94,236 @@ public static void main(String[] args) throws Exception {
             parser.printUsage(System.err);
             return;
         }
-        if (opts.arguments.isEmpty()) {
+
+        if (opts.xqFiles.isEmpty()) {
             parser.printUsage(System.err);
             return;
         }
+
         VXQuery vxq = new VXQuery(opts);
-        vxq.execute();
-        // if -timing argument passed, show the starting and ending times
-        if (opts.timing) {
```

Comment:
I remember seeing the e-mail. I guess this is a much larger question about the REST workflow. I guess this can be added back when someone needs to do performance testing. Does the code still support repeatedExecution? If so, I think this is needed. If not, then lets remove these CLI arguments.